tallycash/extension
# 🦊  Impersonate MetaMask on OpenSea when TallyHo is set as default wallet

Resolves https://github.com/tallycash/extension/issues/790
- Allow OpenSea to connect with TallyHo after click on Metamask icon if TallyHo is set as the default wallet. 
- If TallyHo is not the default wallet clicking on Metamask on OpenSea will open Metamask or its download page.

![image](https://user-images.githubusercontent.com/20949277/169039679-2bf5b96a-7437-4a20-ba00-2009fb4f295a.png)

### Testing

- set TallyHo as default wallet
- try to connect to [OpenSea](https://opensea.io/) via MetaMask icon
- switch off default wallet switcher
- reload OpenSea
- click on MetaMask icon should not open TallyHo

It would be nice to do some regression tests with different dapps we know are working correctly (Uniswap etc)
